R/create.R

```diff
@@ -144,8 +144,8 @@ create_shifted_cases <- function(reported_cases, shift,
 #' Construct the Required Future Rt assumption
 #'
 #' @description `r lifecycle::badge("stable")`
-#' Converts the `future` argument from `rt_opts()` into arguments that can be
-#' passed to `stan`.
+#' Converts the `future` argument from [rt_opts()] into arguments that can be
+#' passed to `{stan}`.
```

Comment:
It's not an r package so do you think this syntax approach is the right?

Reply:
You're right. We should probably reserve this for only R packages. I guess we should not format it all, i.e., 
`{stan}` -> stan.

Reply:
This has been fixed in 5921c926751924ca11f1397baa16582d0e192c12.